compare/commands/compare/interactors/tests/test_fetch_impacted_files.py

```diff
@@ -590,6 +590,7 @@ def test_impacted_files_filtered_by_flags_and_commit_comparison_for_pull(
             CommitFactory(repository=repo),
         )
         pull = PullFactory(
+            pullid=256,
```

Comment:
- Does this mean setting to any ID less than 256 would not work?
- If you add a test (that creates a Pull object in the DB) that runs before this one, does that mean this would fail? Since the next available unused ID is 257?

Reply:
> Does this mean setting to any ID less than 256 would not work?
No, it's just a random number.

> If you add a test (that creates a Pull object in the DB) that runs before this one, 
well the other test I mentioned that causes this issue creates another pull object in the database with ID 44. So, no? 

Where did you get that the next available ID is 257?...

Reply:
Oh ok I didn't know it was a random number, I though anything below 256 is used.